regression/contracts/assigns_enforce_function_calls/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--enforce-contract foo
+^EXIT=1$
+^SIGNAL=0$
+^main.c:\d+:\d+: error: illegal target in assigns clause$
+^CONVERSION ERROR$
+--
+--
+Check that function call expressions are rejected in assigns clauses.
```

Comment:
Missing newline here and a bunch of other ".desc" files.

Reply:
Fixed it.